src/style.css

```diff
@@ -1 +1,70 @@
-/* Styles go here */
+
+* {
+  box-sizing: border-box;
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  color: #000;
+}
+
+.main-header {
+  display: none;
+}
+
+.header {
+  display: flex;
+  max-width: 1200px;
+  justify-content: space-between;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  padding-left: 50px;
```

Comment:
instead of adding padding to left to logo and on the right of last list item add horizontal padding to `.header` class for simple application